tools/write_release_and_log.py

```diff
@@ -0,0 +1,145 @@
+"""
+Standalone script for writing release doc and logs
+-------------------------------------------------
+Cmd -> python write_release_and_log.py <LOG_START> <LOG_END>
+Example -> python write_release_and_log.py v1.6.0 v1.8.0
```

Comment:
One usability question here: with the legacy infrastructure I can do this without tag specifiers during the release process with `paver write_release_and_log`. Is this an intentional change in "ergonomics" / design?

I believe the reason is that we manually edit in values to `pavement.py` during release cycle work.

Reply:
> I believe the reason is that we manually edit in values to pavement.py during release cycle work.

The idea was to eliminate the need to change the variables each time and make the process more intuitive. 
In the current design, the latest Release doc is auto fetched while generating the `Changelog` requires the specific `LOG_START` and `LOG_END` tag specifiers.

Reply:
Indeed. It's also similar to what `tools/authors.py` does.

Reply:
@sayantikabanik can you change the version numbers used in the example though to be more realistic? We'd always want to use consecutive numbers, so `v1.7.0 v1.8.0` here.